certora/specs/CER-1-Owner/CER-59-Owner-override.spec

```diff
@@ -0,0 +1,24 @@
+// CER-59: EVC MUST NOT override the already stored Owner for a 
+// given addressPrefix
+import "../utils/IsMustRevertFunction.spec";
+
+persistent ghost bool overwroteOwner {
+    init_state axiom overwroteOwner == false;
+}
+
+hook Sstore currentContract.ownerLookup[KEY bytes19 address_prefix] address newValue (address oldValue) STORAGE {
+    if (oldValue != 0 && newValue != oldValue) {
+        overwroteOwner = overwroteOwner || true;
```

Comment:
since the assignment is in an if, you can just assign true:
`overwroteOwner =  true;`
alternative, without an if
`overwroteOwner = overwroteOwner || (oldValue != 0 && newValue != oldValue);`